arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5105,6 +5105,9 @@ arangodb::Result ClusterInfo::agencyHotBackupLock(std::string const& backupId,
           builder.add("old", VPackValue("Normal"));
         }
       }
+
+      builder.add(VPackValue(to_string(boost::uuids::random_generator()())));
```

Comment:
You create an RNG here and immediately get a random number from it. I've quickly glanced at the boost documentation and saw that its an (mt19937) PRNG. However, I couldn't immediately find where the seed is coming from, which is important. Do you know the answer to that? I'm asking because:
* the seed might simple, as in a timestamp or even always the same, in which case it's too weak - we could realistically get collisions and maybe have security implications
* the seed might come from a random device and will probably be very expensive to get

Reply:
i do not remember but it is seeded, so that randomness is given.